main.cpp

```diff
@@ -157,6 +159,20 @@ void Octopus::squirtInk(float inkSize, char escapeDirection)
     std::cout << "Ink amount after squirt: " << inkAmount <<std::endl;
 }
 
+float Octopus::huntForFood(int howHungry)
+{
+    for(int i = 0 ; i < howHungry ; ++i )
+    {
+        catchAFish(i, i) ? size += 2.2f : size -= 1.1f ;
```

Comment:
Don't write code like this. 
if `catchAFish()` returns something, then use a correct `if()` to show that. 
```
if( catchAFish(i, i) != 0 )
    size += 2.2f;
else
    size -= 1.1f;
```
- [ ] revise

Reply:
is it okay to do it as
size = catchAFish(i,i) ? size +2.2f : size - 1.1f ;
As you posted in the next comment? When/how do I use ternary operators properly?

Reply:
Of I see...use an If because catchafish returns a bool